style.css

```diff
@@ -1,21 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 html {
   scroll-behavior: smooth;
+  box-sizing: border-box;
+  background-color: #333;
 }
 
 body {
-  margin: 0;
+  box-sizing: border-box;
+  margin: auto;
   height: 100%;
+  width: 100%;
   background-color: white;
+  font-size: 20px;
 }
 
 .navContainer {
-  margin: 20px;
-  margin-top: 40px;
-  padding: 10px;
+  margin-top: 54px;
+  padding: 0 6.4%;
   display: flex;
+  max-width: 100%;
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

There are multiple ways of achieving this, one of them is creating a class `"flex"`:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
Hello @anagudelogu ! I am trying to do this but it breaks every time I move the flex outside of each selector. Any tips?
Thanks a bunch.